bentoml/xgboost.py

```diff
@@ -137,7 +136,7 @@ def save(
         bst = bentoml.xgboost.load("my_xgboost_model:latest") # or
         bst = bentoml.xgboost.load(tag)
     """
-    context = {"xgboost": xgb.__version__, "bentoml_version": BENTOML_VERSION}
+    context = {"xgboost": xgb.__version__}
```

Comment:
what's the reason for removing bentoml version here?

Reply:
we already have a bentoml_version field under the `model_details.yaml`
```yaml
api_version: v1
bentoml_version: 0.13.1+89.g9f78a32.dirty
context:
  xgboost: 1.4.2
created_at: 2021/09/30 04:29:49
labels: []
metadata: {}
module: bentoml.xgboost
name: xgboost_test
options: {}
path: /home/aarnphm/bentoml/models/xgboost_test/20210930_0703E3
tag: xgboost_test:20210930_0703E3
version: 20210930_0703E3

```